src/win/internal.h

```diff
@@ -302,6 +302,7 @@ int uv_translate_sys_error(int sys_errno);
  * FS
  */
 void uv_fs_init();
+typedef int (*uv__fs_mktemp_func)(uv_fs_t* req, size_t num_x, WCHAR *ep);
```

Comment:
Just at the top of the file? There's no other typedef in there, so I'm not sure where this should go.